src/Command/TwoStepVerificationCommand.php

```diff
@@ -16,32 +16,21 @@
 use FOS\UserBundle\Model\UserManagerInterface;
 use Sonata\UserBundle\GoogleAuthenticator\Helper;
 use Sonata\UserBundle\Model\UserInterface;
-use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 
-/**
- * NEXT_MAJOR: stop extending ContainerAwareCommand.
- */
-class TwoStepVerificationCommand extends ContainerAwareCommand
+class TwoStepVerificationCommand extends Command
 {
-    /**
-     * @var ?Helper
-     */
-    private $helper;
+    protected static $defaultName = 'sonata:user:two-step-verification';
 
-    /**
-     * @var ?UserManagerInterface
-     */
+    private $helper;
     private $userManager;
 
-    /**
-     * NEXT_MAJOR: make $helper and $userManager mandatory (but still nullable).
```

Comment:
to make $helper and $userManger mandatory I need to remove this

https://github.com/sonata-project/SonataUserBundle/blob/28d48991fbc31ee5297dca8874e0e86933e8d938/.php_cs.dist#L47

Reply:
Not sure.

This enforce to write `?Foo $foo = null` instead of `Foo $foo = null` but you can still write `?Foo $foo`

Reply:
Yeah That's true!

Thank you